instrumentation/kafka-clients/kafka-clients-0.11/library/src/main/java/io/opentelemetry/instrumentation/kafkaclients/KafkaTracingHolder.java

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.kafkaclients;
+
+import io.opentelemetry.api.GlobalOpenTelemetry;
+
+abstract class KafkaTracingHolder {
+
+  static final KafkaTracing tracing = KafkaTracing.create(GlobalOpenTelemetry.get());
+}
```

Comment:
this class seems unnecessary, i'd suggest removing it, it's ok to instantiate two `KafkaTracing` instances for the two subclasses (and ideally even better if user can pass in their own `KafkaTracing` to those two subclasses)